### ISSUE TITLE
Print tidal version

### ISSUE THREAD

**Issue by ndr-brt** [tidalcycles/atom-tidalcycles#147]
Could be useful to print tidal version on the console, using the `ghc-pkg latest tidal` command

**Comment by yaxu** [comment]
Yes that would be great! I think easier to do it by sending `tidal_version` to ghci, a variable which tidal provides

**Comment by yaxu** [comment]
Perhaps we could just put something like `putStrLn $ "[TidalCycles v" ++ tidal_version ++ "]"` in the default BootTidal.hs ?

**Comment by ndr-brt** [comment]
Seems fair! I already opened a pr, but this seems easier
